utils/gen-corim/README.md

```diff
@@ -0,0 +1,41 @@
+# CoRIM Generation
+
+## Installing and configuring
+
+To install the `gen-corim` command, do:
+
+```
+$ go install github.com/veraison/services/gen-corim@latest
+```
+
+## Usage
+
+```
+$ gen-corim --evidence-file=evidence.cbor --attest-scheme=psa --key-file=key.json --template-dir=templates (--corim-file=endorsements/output.cbor)
+```
+
+On success, you should see something like this printed to stdout:
+
+```
+>> generated "endorsements/output.cbor" using "evidence.cbor"
+```
+### Supplied Arguments
+#### Evidence File
+
+CBOR-encoded evidence token to be used via the `--evidence-file` switch (or equivalently its `-e` shorthand).
+
+### Attestation Scheme
+
+The attestation scheme to be used via the `--attest-scheme` switch (abbrev. `-a`). The only attestation schemes supported by this service are `psa` and `cca`.
+
+### Key File
+
+Public key material needed to verify the evidence via the `--key-file` switch (abbrev. `-k`). The key file is expected be in [jwk](https://openid.net/specs/draft-jones-json-web-key-03.html) format.
+
+### Template Directory
+
+The directory containing the CoMID and CoRIM templates via the `--template-dir` switch (abbrev. `-t`). This directory must exist and must contain files named `comid-template.json` and `corim-template.json` which contain the respective templates. Some examples of CoMID and CoRIM JSON templates can be found in the [data/templates](data/templates) folder.
```

Comment:
There are already good number of arguments and supplying more files as input would complicate the argument list even further.

I quite like the idea of the default location of `$PWD/templates/` , if the directory argument is  not supplied by the user.
However the Readme should prescribe file names as done in the PR

Reply:
Well, the whole point is to avoid supplying them all the time (these should be optional), but to allow them to be overwritten easily. Chances are that the same corim templame will be re-used (as it contains general info) for multiple comids. So I want to be able to quicly generate tokens for different comids without having to setup template directories for each one (and needlessly duplicating the corim template in each).

Reply:
Sure we can make the supplied directory as `optional`  and expect the test to fail if:
1) Either `$PWD/templates/ ` is missing
OR
On the provided path above, the prescribed files: `comid-template.json` and `corim-template.json` missing.